frontend/app/api/questions/[id]/route.ts

```diff
@@ -5,10 +5,9 @@ export async function PUT(req: Request, { params }: { params: { id: number } })
     const body = await req.json();
     const question = await UpdateQuestion.call(params.id, body.answerContent);
 
-    return Response.json({ data: question }, { status: 200 });
+    return Response.json(question, { status: 200 });
```

Comment:
Are we not keeping the `data` object?

Reply:
Yes we are. Sorry I messed the merge because there were changes all over the place because of the prettier formatting

Reply:
I pushed a fix for this. I also added it to the airstack endpoints